app/src/ui/changes/commit-message.tsx

```diff
@@ -139,10 +139,10 @@ export class CommitMessage extends React.Component<
 
     if (!shallowEquals(prevProps.commitMessage, this.props.commitMessage)) {
       if (this.props.commitMessage) {
-        this.setState({
-          summary: this.props.commitMessage.summary,
-          description: this.props.commitMessage.description,
-        })
+        this.setState((_state, props) => ({
+          summary: props.commitMessage.summary,
```

Comment:
This one is blowing up because we previously checked `this.props.commitMessage` was not null.

I think a change like this is what's needed (which bypasses needing parameters inside `setState`):

```diff
@@ -138,10 +138,11 @@ export class CommitMessage extends React.Component<
     }

     if (!shallowEquals(prevProps.commitMessage, this.props.commitMessage)) {
-      if (this.props.commitMessage) {
+      const { commitMessage } = this.props
+      if (commitMessage) {
         this.setState({
-          summary: this.props.commitMessage.summary,
-          description: this.props.commitMessage.description,
+          summary: commitMessage.summary,
+          description: commitMessage.description,
         })
       } else {
         this.setState({ summary: '', description: null })
```